socketio_middleware/src/socketio.rs

```diff
@@ -53,7 +53,7 @@ pub async fn broadcast(room_id: &str, event: &str, message: &str) {
                         event.to_string(),
                         message.to_string(),
                     )));
-                    debug!("Found socketid {} in room {}, sending message = {}", channel.sid(), room_id, message);
+                    trace!("Found socketid {} in room {}, sending message = {}", channel.sid(), room_id, message);
```

Comment:
Keep as debug unless we have a rule as to when we use trace vs debug.

Reply:
NP